packages/coinstac-ui/app/render/components/results/result.jsx

```diff
@@ -41,46 +52,353 @@ class Result extends Component {
       });
   }
 
+  drawScatter() {
```

Comment:
I know it's a bit of a hassle, but it'll be much more manageable going forward if we move the various plots into their own components and just import them into `results.jsx`. Maybe we could make a subdirectory in `results` called `graphs` or `plots`? Then put `box-plot.jsx`, `scatter-plot.jsx` and `results-table.jsx` components in there?